src/cli/idm/idm.js

```diff
@@ -0,0 +1,28 @@
+import { Command } from 'commander';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+export default function setup() {
+  const program = new Command('idm')
+    .helpOption('-h, --help', 'Help')
+    .description('Manage IDM configuration.')
+    .executableDir(__dirname);
+
+  program
+    .command('list', 'List all IDM configuration objects.')
+    .showHelpAfterError();
+
+  // Do all 3 commands (raw / all / one ) or seperate?
+  program
+    .command('export', 'Export IDM configuration objects.')
+    .showHelpAfterError();
+
+  program
+    .command('count', 'Count number of managed objects of a given type.')
```

Comment:
Minor but I propose a simplified wording:
'Count managed objects.'

Reply:
Do we still need the count (now that we have a dashboard showing counts)? If you think its still useful, we should probably use the metrics API that the dashboard uses.